test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -1490,7 +1490,7 @@ describe('Test gl3d relayout calls', function() {
         .then(done);
     });
 
-    it('@gl should maintain projection type when resetCamera buttons clicked after switching projection type from perspective to orthographic', function(done) {
+    it('@gl resetCamera buttons should be able to reset projection type after switching projection type from perspective to orthographic', function(done) {
```

Comment:
Why do you change your mind here?

Reply:
Now that we could switch projections to initial projection I thought we may change this.
If the user is not happy with the changes made to the camera and wants to get back to initial/home view this allows that. [Codepen](https://codepen.io/MojtabaSamimi/pen/JzbRjb?editors=0010).